gnome-initial-setup/gis-driver.c

```diff
@@ -370,6 +370,7 @@ gis_driver_is_in_demo_mode (GisDriver *driver)
 
 #define DEMO_ACCOUNT_USERNAME "demo-guest"
 #define DEMO_ACCOUNT_FULLNAME "Demo"
+#define DEMO_ACCOUNT_AVATAR_IMAGE "/usr/share/pixmaps/faces/sunflower.jpg"
```

Comment:
Is this not defined somewhere else? I would imagine it is done for regular users too?

Reply:
Would it make sense to share the define?

Reply:
I guess we could move the define to "gnome-initial-setup/gnome-initial-setup.h" since that is included in both places.